android-client-sdk/src/main/java/com/devcycle/sdk/android/eventsource/EventParser.java

```diff
@@ -290,11 +289,11 @@ private void eventTerminated() throws UnsupportedEncodingException {
   
   private void dispatchMessage(MessageEvent message) {
     try {
-      Timber.d("Dispatching message: {}", message);
+      Timber.d("Dispatching message: %s", message);
       handler.onMessage(message.getEventName(), message);
     } catch (Exception e) {
       Timber.w("Message handler threw an exception: " + e.toString());
-      Timber.d("Stack trace: {}", new LazyStackTrace(e));
+      Timber.e("Stack trace: %s", new LazyStackTrace(e));
```

Comment:
ah, that makes sense. I thought this might have just been an error I made earlier when converting from the LD logger to Timber, but I think youre right